sdk/servicebus/service-bus/tests.yml

```diff
@@ -27,4 +27,19 @@ jobs:
         AAD_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AAD_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AAD_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        SERVICEBUS_CONNECTION_STRING: $(service-bus-test-connection-string)
+        AZURE_SUBSCRIPTION_ID: $(AZURE_SUBSCRIPTION_ID)
+        SERVICEBUS_CONNECTION_STRING: $(SERVICEBUS_CONNECTION_STRING)
+        RESOURCE_GROUP: $(RESOURCE_GROUP)
+        QUEUE_NAME: "partitioned-queue"
```

Comment:
none of QUEUE_NAME and below are required to be included here, since the CI would depend on the default values that we specify in code (test utils).
The values used here are for local testing and shouldn't really be used anywhere else.

Reply:
According to @ramya-rao-a , the browser tests need these resources. Is that not true?

Reply:
The node tests are able to create the required queues/topics/subscriptions, but the browser tests dont do this.
Therefore, if we are planning to use an arm template to create a Service Bus namespace for each test run, we need it to create the queues/topics/subscriptions as well.

But the question is whether these environment variables need to be listed in the tests.yml
The answer is yes if we are using names that are uniquely generated for each test run.
But, if the arm template is using a given set of names, then we don't need to have these env variables set here.

Reply:
oh ok. I'll remove these env variables